components/tidb_query/src/batch/executors/index_scan_executor.rs

```diff
@@ -183,9 +182,10 @@ impl ScanExecutorImpl for IndexScanExecutorImpl {
                 // This is a unique index, and we should look up PK handle in value.
 
                 // NOTE: it is not `number::decode_i64`.
-                value
-                    .read_i64::<BigEndian>()
-                    .map_err(|_| other_err!("Failed to decode handle in value as i64"))?
+                (value
+                    .read_u64()
```

Comment:
`read_i64()` exists here, why don't we still using it?

Reply:
@mapleFU Take a look at what `read_i64()` does and you will know.